emuDCS/PeripheralCore/src/common/TMB.cc

```diff
@@ -5732,11 +5873,16 @@ void TMB::SetTMBRegisterDefaults() {
   // 0X14C - 0X158 = ADR_V6_GTX_RX[CFEB]: GTX link control and monitoring
   //---------------------------------------------------------------------
   for (int i=0; i < 7; i++) {
-      gtx_rx_enable_[i] = gtx_rx_enable_default;
-      gtx_rx_reset_[i] = gtx_rx_reset_default;
-      gtx_rx_prbs_test_enable_[i] = gtx_rx_prbs_test_enable_default;
+    gtx_rx_enable_[i] = gtx_rx_enable_default;
+    gtx_rx_reset_[i] = gtx_rx_reset_default;
+    gtx_rx_prbs_test_enable_[i] = gtx_rx_prbs_test_enable_default;
   }
   //
+  //---------------------------------------------------------------------
+  // 0X15C ADR_V6_CFEB_BADBITS_CTRL: CFEB Bad Bits Control/Status (See Adr 0x122) (extra DCFEB Bad Bits on OTMB)
+  //---------------------------------------------------------------------
+  dcfeb_badbits_reset_ = dcfeb_badbits_reset_default;
+  dcfeb_badbits_block_ = dcfeb_badbits_block_default;
```

Comment:
the right hand side dcfeb_badbits_block_default and dcfeb_badbits_reset_default are not defined ==> doesn't compile